doc/CEP.md

```diff
@@ -0,0 +1,260 @@
+# Complex Event Processing (CEP) example
+
+This document explains the configuration and use of the CEP within the scope of Fiware Tourguide.  For a detailed explanation of CEP, please read the official documentation available at https://proactive-technology-online.readthedocs.org/en/latest/index.html.
```

Comment:
do you really need the phrase 'scope of Fiware Tourguide' ? I would drop that sentence